### PR TITLE
[tests-only] run tus test for file with invalid-name

### PR DESCRIPTION
**Pull request by swoichha**
to run TUS tests for file with invalid-name, see https://github.com/owncloud/core/pull/38217 and https://github.com/owncloud/core/pull/38226

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -16,8 +16,8 @@ config = {
     'onlyoffice':'frontend'
   },
   'apiTests': {
-    'coreBranch': 'master',
-    'coreCommit': '27eaa32d777052d1d210647cd38895e647c48fed',
+    'coreBranch': 'changes-tusTests-invalid-file-names',
```
Comment (phil-davis) [inline .drone.star:19]: We can't merge like this. We need a "general" PR to OCIS that updates the coreCommit for master and has all the needed expected-failures changes for whatever is now merged in core master.

**Review by phil-davis — CHANGES_REQUESTED** [review]

File: .drone.star (hunk 1 above)
Comment (swoichha) [reply .drone.star:19]: I will change it back to `'coreBranch': 'master'` once this PR gets merged https://github.com/owncloud/core/pull/38226

**Review by phil-davis — APPROVED** [review]
LGTM - brings expected-failures up-to-date with the tests merged in core master up to about 30 minutes ago.

**Comment by phil-davis** [comment]
I will make a similar  PR to `cs3org/reva`

**Pull request merged**